axisregistry/volume.textproto

```diff
@@ -0,0 +1,14 @@
+tag: "VOLM"
+display_name: "Volume"
+min_value: 0.0
+default_value: 0.0
+max_value: 100.0
+precision: 1
+fallback {
+  name: "Normal"
+  value: 0.0
+fallback {
+  name: "Loud"
```

Comment:
@RosaWagner, you need to add closing curly bracket here before the second `fallback` keyword. Otherwise it breaks the Protobuf parser.

I noticed this typo when integrating these files to the FontBakery GFAxisRegistry-related checks at https://github.com/googlefonts/fontbakery/pull/3057

Reply:
Thanks @felipesanches for the review 👍